src/main/host/process.rs

```diff
@@ -694,7 +694,7 @@ impl Process {
         let mut host_shmem_protected = host.shim_shmem_lock_borrow_mut().unwrap();
         let threads = self.threads.borrow();
         for thread in threads.values() {
-            let mut thread = thread.borrow_mut(host.root());
+            let thread = thread.borrow_mut(host.root());
```

Comment:
Should the be renamed to just `borrow()`?

Reply:
This line is removed in a later commit